.github/actions/derive-frontend-bucket/action.yaml

```diff
@@ -0,0 +1,30 @@
+name: "Derive bucket name for SPA action"
+description: "Derives the name of the bucket used to store the SPA for subsequent actions to run against"
+
+inputs:
+    environment:
+        description: "The name of the environment to action the infrastructure"
+        required: true
+    workspace:
+        description: "The name of the workspace to action the infrastructure into."
+        required: true
+    repo_name:
```

Comment:
We usually refer to 'repo_name' as the 'domain'. (See deploy-infrastructure.yaml). Few occurrences of this in various actions and workflows

Reply:
repo_name changed to domain